openverse_catalog/dags/providers/provider_api_scripts/phylopic.py

```diff
@@ -111,18 +120,24 @@ def _get_total_images():
     return total
 
 
-def _create_endpoint_for_IDs(**args):
+def _create_endpoint_for_IDs(**kwargs):
     limit = LIMIT
 
-    if args.get("date"):
-        # Get a list of objects uploaded/updated on a given date.
-        date = args["date"]
-        endpoint = f"http://phylopic.org/api/a/image/list/modified/{date}"
+    if (date_start := kwargs.get("date_start")) is not None and (
+        date_end := kwargs.get("date_end")
+    ) is not None:
```

Comment:
I know this file is linted so definitely feel free to ignore this, but this formatting looks strange to me -- at a glance I read the final paren on L128 as closing the initial paren on L126 😕

Reply:
Yea...`black` is not being super helpful here :disappointed: I'll see if I can move things around to make it a little clearer.

Reply:
Done in [3d5e27a](https://github.com/WordPress/openverse-catalog/pull/423/commits/3d5e27aa5211291e26f09fb3af65ede1ca2d9fc4)!

Reply:
nice fix, that was really strange to try and read!